core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -231,6 +236,14 @@ public ListBucketResult call() throws S3Exception {
         final FileSystem fs = getFileSystem(authorization);
         checkPathIsAlluxioDirectory(fs, path);
 
+        if (tagging.isEmpty()) {
```

Comment:
I'm not sure how this works. If the request does set the tagging parameter, what does this tagging variable become?

Reply:
It becomes the default value which is "null". The reason why it's setup this way is because the way this gets sent from the client is ?tagging without any value.